package-lock.json

```diff
@@ -1,1161 +1,8 @@
 {
   "name": "cardano-multiplatform-lib",
   "version": "3.1.1",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
Depending on which version of NPM you have installed, it will change the lockfileVersion. It's annoying because if you have multiple people on your team with different NPM versions, it causes issues. Maybe v2 has been out long enough that it's less of an issue now, but still annoying.

Reply:
```
> npm -v
8.5.4
```

Reply:
you need to undo your changes to `package-lock.json` and then run npm install again making sure you're on a recent enough version that supports the v2 format (8.5.4 should be recent enough)

Reply:
Kk. Ran that. I'm confused why it acted differently this time. I didn't change my `npm` version. Maybe it was another script that messed it up, like `rust:build-nodejs` maybe.